src/deriveLinkDefaults.ts

```diff
@@ -156,7 +156,13 @@ export function initialLoader(datasets: ILinkConfig['datasets']) {
     return Promise.all(resolvedDataSets).then((data) => {
       const cleaned = data.filter((d): d is DataSet => d != null);
       cleaned.forEach((d) => {
-        if (d.params && !Array.isArray(d.params) && d.params._endpoint) {
+        console.log(d)
+        console.log(JSON.stringify(d))
```

Comment:
At this point in the execution, `d.params` no longer includes a number of the keys+values that are passed in via the base64-encoded url string, as you might expect...  Its because (as best as i can tell) only the parameters that match function argument names are allowed through [the call in `loadImpl()`](https://github.com/cmu-delphi/www-epivis/blob/781ff27e37dc1bb7e67cdea11ec8c906699d34c2/src/deriveLinkDefaults.ts#L126) that sneakily proxies to [an appropriate `importXYZendpoint()` method](https://github.com/cmu-delphi/www-epivis/blob/781ff27e37dc1bb7e67cdea11ec8c906699d34c2/src/deriveLinkDefaults.ts#L47-L63).  However, `_endpoint` is added back to the `params` map [where `loadDataSet()` calls `loadEpidata()`](https://github.com/cmu-delphi/www-epivis/blob/a22b65c85b7097968c5fdc647ba9eae322aa4766/src/api/EpiData.ts#L150).

The current state of code in this PR might work work if we modify the kludge `patchDataSet()` above to accept another argument for a custom title that can then be added back into the `.params` of the `DataSet` returned (it is already populating the `.color` attribute).  A less hacky way to handle this would be to make the custom title into a proper first-order member of the `DataSet` class, the way `.color` and `.params` are -- but so much of this code is already unsightly, i dont know how much i care about that.